src/main/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResource.java

```diff
@@ -268,6 +271,7 @@ public Uni<RestResponse<CommandResult>> postCommand(
                         .withEmbeddingProvider(embeddingProvider)
                         .withCommandName(command.getClass().getSimpleName())
                         .withRequestContext(requestContext)
+                        .withMeterRegistry(meterRegistry)
```

Comment:
this should be set on the contextBuilderSupplier in the ctore for this class, it must be set in every Resource, and it must be non null